cmake/Hunter/config.cmake

```diff
@@ -121,12 +121,15 @@ set(opencv_cmake_args
   BUILD_opencv_dnn=OFF #  -/-
   BUILD_LIBPROTOBUF_FROM_SOURCES=NO
 
-  BUILD_opencv_imgproc=ON   # required for flow
-  BUILD_opencv_optflow=ON   # optflow
-  BUILD_opencv_plot=ON      # tracking -> plot
-  BUILD_opencv_tracking=ON  # tracking
-  BUILD_opencv_video=ON     # tracking -> video
-  BUILD_opencv_ximgproc=ON  # optflow -> ximgproc
+  # imgproc required for flow
+  # tracking -> plot
+  # tracking -> video
+  # optflow -> ximgproc
+  # highgui: dependency for acf
+  # videostab: needed for drishti/core/padding.cpp
+  # objdetect: needed for drishti/face/FaceDetector.h
+  # videoio: needed for drishti/src/app/videoio/VideoSourceCV.cpp
+  BUILD_LIST=imgproc,optflow,plot,tracking,video,ximgproc,highgui,videostab,objdetect,videoio
```

Comment:
We can simplify this slightly for the default configuration.  Probably closer to this:

```
BUILD_LIST=core,imgproc,videoio,highgui,videostab,objdetect
```

Where `videostab` and `objdetect` can probably be avoided with a little work.  I'll test the reduced list.

Reply:
We could also trim `webp` and `openexr` for the default builds.

Reply:
`ocv_define_module(videostab opencv_imgproc opencv_features2d opencv_video opencv_photo opencv_calib3d ... )`

For reference, the current minimal use of `videostab` introduces a number of additional opencv module depepdencies.

It is used here:

src/lib//drishti/core/padding.cpp:    cv::videostab::ColorAverageInpainter
src/lib//drishti/face/face_util.cpp:cv::Mat estimateGlobalMotionLeastSquares

These calls can be replaced with a little work.